packages/trader/src/sass/app/modules/contract/accumulators-stats.scss

```
@@ -73,7 +72,8 @@
         display: flex;
         align-items: center;
         padding: 0 3.2rem 0 1.6rem;
-        justify-content: flex-start;
+        width: 100%;
+        justify-content: space-between;
```

@yauheni-kryzhyk-deriv as discussed with designers, it should be flex-start:
```suggestion
        justify-content: flex-start;
```
added it in this [commit](https://github.com/maryia-binary/deriv-app/commit/da1a52cecc2d93a0f4b657a8c5cb9028b2ee3bab)

---

@maryia-binary but it looks inappropriate, specially  on mobile version.  there is to big gap between last digit and arrow